### ISSUE TITLE
Adapt to Nerd Font 3.0

### ISSUE THREAD

**Issue by Saafo** [jesseduffield/lazygit#2603]
**Describe the bug**

Some icon can't be correctly displayed after upgrade nerd font to 3.0

**To Reproduce**

1. `showIcons = true`
2. using a 3.0 nerd font
3. icons can't be correctly displayed

**Expected behavior**

**Screenshots**
<img width="47" alt="image" src="https://user-images.githubusercontent.com/46736350/236617555-feae3437-8554-4c13-8e45-fff697415a13.png">

**Version info:**

commit=, build date=, build source=homebrew, version=0.38.2, os=darwin, arch=arm64, git version=2.40.1

**Additional context**
Maybe there's some breaking changes on [Nerd Font 3.0](https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.0.0)

**Comment by latipun7** [comment]
I guess this is already done by https://github.com/jesseduffield/lazygit/pull/2731

**Comment by Saafo** [comment]
> I guess this is already done by #2731

Yes, @jesseduffield I wonder if you have any plan to release a new version?

**Comment by jesseduffield** [comment]
@Saafo new release is blocked by https://github.com/jesseduffield/lazygit/issues/2744
